samples/key_value_api/docker/Dockerfile.x86

```diff
@@ -0,0 +1,3 @@
+FROM ghcr.io/insight-platform/savant-deepstream:latest
+
+COPY scripts/ /scripts/
```

Comment:
You can simply mount the scripts, which will prevent you from building a separate docker.